packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1130,23 +1130,24 @@ The performance optimization will only be applied when all three parameters are
 
 | Parameter   | Description                                                                                                                                    | Param Type   | DataType   | Required |
 | :---------- | :--------------------------------------------------------------------------------------------------------------------------------------------- | :----------- | :--------- | :------- |
-| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties                                                                      | Query        | String     | FALSE    |
-| filter      | [Filters](/docs/reference/core-okta-api/#filter) users with a supported expression for a subset of properties                  | Query        | String     | FALSE    |
-| search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties          | Query        | String     | FALSE    |
+| search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties. Prefer this option for optimal performance         | Query        | String     | FALSE    |
 | limit       | Specifies the number of results returned (maximum 200)                                                                                         | Query        | Number     | FALSE    |
 | after       | Specifies the pagination cursor for the next page of users                                                                                     | Query        | String     | FALSE    |
+| filter      | [Filters](/docs/reference/core-okta-api/#filter) users with a supported expression for a subset of properties                  | Query        | String     | FALSE    |
 | sortBy      | Specifies field to sort by (for search queries only)                                                                                           | Search query | String     | FALSE    |
 | sortOrder   | Specifies sort order asc or desc (for search queries only)                                                                                     | Search query | String     | FALSE    |
+| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties (but prefer search; see above)                       | Query        | String     | FALSE    |
 
   * If you don't specify a value for `limit`, the maximum (200) is used as a default.  If you are using a `q` parameter, the default limit is 10.
   * An HTTP 500 status code usually indicates that you have exceeded the request timeout. Retry your request with a smaller limit and [paginate](/docs/reference/core-okta-api/#pagination) the results.
+  * We strongly advise that where possible, you prefer the `search` parameter, which delivers optimal performance. The `q` or `filter` parameters may struggle to perform, in which case we advise reformatting the request to use `search`.
```

Comment:
I don't think "we" use "we" in our docs.  Try: 
Where possible, prefer the `search` parameter, which delivers the best performance. If a `q` or `filter` parameter is responding too slowly, try reformatting the request to use `search`.

Reply:
"Found occurrences: `okta-developer-docs`: 642 occurrence [sic]".  (No, IntelliJ doesn't pluralize 'occurrences'.)

We most certainly do :trollface:

Reply:
However after going through _all_ the comments, I see one from a doc person confirming that, in fact, while it might be used all over, they _would_ prefer to avoid it.  So avoid it We shall.